### DISCUSSION TITLE
Unable to install oreka with Ubuntu

### DISCUSSION THREAD

**Discussion by yacinus** [voiceip/oreka#120] [Q&A] [answered]
Hello,
I deployed asterisk on a virtual machine and now I need to install oreka in order to record the calls and then I prepared a separate Ubuntu server for it
I'm stuck on the step of deploying orktrack.war to tomcat webapps. I couldn't find any file orktrack.war.
Same thing for orkweb.war.
Is there a procedure of compiling which will generate the files? If so, please let me know ☺️
I hope to see an answer soon.
Thank you. 😊

**Comment by kingster** [comment, marked as answer]
Please download the files from the releases section.

https://github.com/voiceip/oreka/releases
